### ISSUE TITLE
human-readable datetime

### ISSUE THREAD

**Issue by keikoro** [django-webpack/webpack-bundle-tracker#65]
Would it be possible to allow different formatting options for the `startTime` and `endTime` values? I find myself needing to check them while experimenting with webpack settings but unix timestamps aren't really that user-friendly. An output of `HH:mm:ss` would be more helpful.

(Though I'm not sure if `toLocaleTimeString()` would be best-suited or if it should be the only way to display the datetimes in a more human-readable format. It'd easily work fine locally but might require more fiddling when used on servers or in containers etc.)

**Comment by propkitty** [comment]
Couldn't agree more with @keikoro - this would be magnitudes better with a human-readable datetime. I end up having to copy and paste into something that 'translates' which shouldn't be necessary.

**Comment by fjsj** [comment]
Yes, it would be great to use ISO 8601 formatted dates there. But I don't have the bandwidth to work on that right now. PRs are welcome!
